benefits/core/templates/core/widgets/verifier_radio_select.html

```diff
@@ -1,5 +1,14 @@
+{% load i18n %}
 
 {% with id=widget.attrs.id %}
+
+  <label for="{{ id }}">
+    <h2 class="pb-4">
```

Comment:
We want to avoid putting headings inside labels: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label#headings

Reply:
done